packages/git-unix/git-unix.3.7.0/opam

```diff
@@ -33,7 +33,7 @@ depends: [
   "astring" {>= "0.8.5"}
   "awa" {>= "0.0.5"}
   "mirage-time"
-  "mirage-unix" {>= "4.0.0"}
+  "mirage-unix" {>= "4.0.0" & < "5.0.0"}
```

Comment:
Can you explain why this is fixing the CI issue? What version of `mirage-unix` was installed?

Reply:
5.0.0. See https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/5471602af769ee147301ec26651b62469f1e1fd2/variant/opam-2.1,compilers,4.13,functoria-runtime.4.0.0~beta3,revdeps,irmin-unix.2.10.0

Reply:
I will fallback with another release of `git` after this one in anyway - I already planned a release of `git`.

Reply:
and it wasn’t detected in https://github.com/ocaml/opam-repository/pull/20605 because at the time, none of its revdeps were installable with mirage >= 4.0

Reply:
Yeah, that's what I thought

Reply:
haaa so mirage-unix.5 is need for mirage.4 (and not mirage-unix.4). These numbers are a bit confusing but I get it now :-)